src/OrbitGl/GpuDebugMarkerTrack.cpp

```diff
@@ -26,14 +26,19 @@ using orbit_client_protos::TimerInfo;
 
 GpuDebugMarkerTrack::GpuDebugMarkerTrack(CaptureViewElement* parent, TimeGraph* time_graph,
                                          orbit_gl::Viewport* viewport, TimeGraphLayout* layout,
-                                         OrbitApp* app,
+                                         uint64_t timeline_hash, OrbitApp* app,
```

Comment:
I wonder if we can get rid of hash here... but probably outside of scope of this PR.

Reply:
Gpu tracks (and subclasses) are identified by timeline_hash. I prefer to have it inside the constructor of the classes. What do you think that could be better instead?